layers/ubuntu1604/java/revisions.bzl

```diff
@@ -16,21 +16,21 @@
 # For more details, check the deps_spec.yaml file in the current folder.
 
 OPENJDK_INSTALLER = struct(
```

Comment:
@alex1545  it looks like it failed to fetch one of the java 11 files. The file name pattern seems changed:

gs://bazel-mirror/openjdk/azul-zulu11.2.3-jdk11.0.1/zulu11.2.3-jdk11.0.1-linux_x64.tar.gz
vs
gs://bazel-mirror/openjdk/azul-zulu10.2+3-jdk10.0.1/zulu10.2+3-jdk10.0.1-linux_x64-allmodules.tar.gz

Also I don't think we should migrate to java 11 yet. Let's somehow turn off the update for OPENJDK_INSTALLER and OPENJDK_SRC? If that's not currently supported. Can we add a feature to it such that we can turn on and off updates for anyone specified in the dep_specs.yaml file (https://github.com/GoogleCloudPlatform/layer-definitions/blob/master/layers/ubuntu1604/java/deps_spec.yaml) ?

Reply:
Even thought the file name pattern (more specifically the version format) has slightly changed, the service is still able to find the correct location of the latest version. The issue is that the 
gs://bazel-mirror/openjdk/azul-zulu11.2.3-jdk11.0.1/zulu11.2.3-jdk11.0.1-linux_x64-allmodules.tar.gz
file that we attempt to download [here](https://github.com/GoogleCloudPlatform/layer-definitions/blob/496e0c889f507bbc2a5b75b16494dca5dbf1663b/layers/ubuntu1604/java/deps.bzl#L36) is not present in the bucket.

And since we want to still stay with java 10 here, I think the best thing to do is just look for the latest java 10 version.

I'll send a PR for that and trigger the service one more time.

Reply:
Thanks!